MIA.cpp

```diff
@@ -84,6 +92,9 @@ try
 		return 1;
 	}
 
+	if (config.verbose)
```

Comment:
I don't really think it's necessary, even in verbose.

Reply:
You can work around that but I believie it's a nice touch.